addons/source-python/packages/source-python/listeners/tick.py

```diff
@@ -109,19 +109,17 @@ def add(self, delay):
 class Delay(WeakAutoUnload):
     """Execute a callback after a given delay."""
 
-    def __init__(
-        self, delay, callback, cancel_on_map_end=False, *args, **kwargs
-    ):
+    def __init__(self, delay, callback, cancel_on_map_end=False, args=(), kwargs={}):
```

Comment:
Why is it a bad practice though? I do agree that it's a minor downside that someone else might copy paste my code and thus have a bug in their code, but I'm not going to write my code based on what someone else might copy paste from it.